### ISSUE TITLE
Description

### ISSUE THREAD

**Issue by jarrodmillman** [scientific-python/scipy.org-new#8]
We need to change
```
      description: Why NumPy? Powerful n-dimensional arrays. Numerical computing tools. Interoperable. Performant. Open source.
```
in `config.yaml`.

Where is it used?

**Comment by rossbar** [comment]
Looking at the generated html for the `numpy.org` site, this description shows up in a meta tag in the header so, AFAICT, does not actually correspond to any element that is actually visible on the page. Maybe it's relevant for search engines or other crawlers, but I suspect you could remove `description:` entirely without any noticeable effect.

**Comment by rossbar** [comment]
I think this one was closed by #13
